protocol/communities/manager.go

```diff
@@ -1530,6 +1600,157 @@ func (m *Manager) IsSeedingHistoryArchiveTorrent(communityID types.HexBytes) boo
 	return ok && torrent.Seeding()
 }
 
+func (m *Manager) DownloadHistoryArchivesByMagnetlink(communityID types.HexBytes, magnetlink string) ([]string, error) {
+
+	id := communityID.String()
+	ml, err := metainfo.ParseMagnetUri(magnetlink)
+	if err != nil {
+		return nil, err
+	}
+
+	m.logger.Debug("Adding torrent via magnetlink for community", zap.Any("id", id), zap.Any("magnetlink", magnetlink))
```

Comment:
You can use strings for zap `zap.String("id", id.String())` or `zap.String("magnet", magnetlink.String()), so you get the hexencoded version (otherwise is going to be base encoded)